marketdata/rest.go

```diff
@@ -996,10 +1117,25 @@ func (c *Client) GetLatestCryptoBar(symbol string, req GetLatestCryptoBarRequest
 	return &bar, nil
 }
 
+// GetLatestCryptoPerpsBar returns the latest bar for a given crypto symbol
+func (c *Client) GetLatestCryptoPerpsBar(symbol string, req GetLatestCryptoBarRequest) (*CryptoPerpsBar, error) {
```

Comment:
`GetLatestCryptoPerpsBars` for multiple symbols should also be added.

Reply:
Done.